src/blockdata/transaction.rs

```diff
@@ -451,7 +451,7 @@ impl Transaction {
 
     /// Is this a coin base transaction?
     pub fn is_coin_base(&self) -> bool {
```

Comment:
Can you add a reference to document below?

https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/fix_transaction_malleability.md#tapyrus-transaction-structure

Reply:
Update comments at c27972a1e3fdcf11e69946a8c6a52a608c27fe84